lib_test/test_client.ml

```diff
@@ -168,6 +168,111 @@ let test_https _ () =
     response;
   Helper_server.teardown server
 
+let test_https_server_certs _ () =
+  let* server = Helper_server.listen ~http_port:8080 () in
+  (* Verify server cert from file *)
+  let* response =
+    Client.Oneshot.get
+      ~config:
+        { Config.default with
+          follow_redirects = true
+        ; max_redirects = 1
+        ; allow_insecure = false
+        ; max_http_version = Versions.HTTP.v1_1
+        ; cacert = Cert.Filepath("./certificates/ca.pem")
+        }
+      (Uri.of_string "https://localhost:9443")
+    in
+  let response = Result.get_ok response in
+  Alcotest.check
+    response_testable
+    "expected response"
+    (Response.create 
+      ~version:Versions.HTTP.v1_1 
+      ~headers:Headers.(of_list [ Well_known.content_length, "1" ])
```

Comment:
is content length necessary here if you never read the response body in the test? Could it just be 0?

Reply:
Yes, response_testable checks the headers and it's returning 1 in content_length